src/utils/FlowYamlUtils.ts

```diff
@@ -123,10 +123,11 @@ function getSectionFromDocument({yamlDoc, section}:
     return sectionNode?.value;
 }
 
-function getPathFromId({node, id} : {
+function getPathFromId({node, id, key} : {
```

Comment:
As GH said below, unused `key` variable, might be removed, or, might be used somewhere down there, if that's what it's meant for.